src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -182,6 +176,28 @@ public Decimal(double value)
             DecCalc.VarDecFromR8(value, out AsMutable(ref this));
         }
 
+        private Decimal(SerializationInfo info, StreamingContext context)
```

Comment:
Do we have a Decimal serialization test?

Reply:
Yep, we have number of cases for decimal in https://raw.githubusercontent.com/dotnet/runtime/master/src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs . They were failing before the fix.